src/umpire/Replay.cpp

```diff
@@ -5,50 +5,45 @@
 // SPDX-License-Identifier: (MIT)
 //////////////////////////////////////////////////////////////////////////////
 
-#include <iostream>   // for std::cout, std::cerr
-#include <stdlib.h>   // for getenv()
+#include <stdlib.h> // for getenv()
+
+#include <iostream> // for std::cout, std::cerr
 
 #if !defined(_MSC_VER)
-#include <strings.h>  // for strcasecmp()
-#include <unistd.h>   // getpid()
+#include <strings.h> // for strcasecmp()
+#include <unistd.h>  // getpid()
 #else
 #include <process.h>
 #define strcasecmp _stricmp
 #define getpid _getpid
 #endif
 
 #include "umpire/Allocator.hpp"
+#include "umpire/Replay.hpp"
 #include "umpire/strategy/AllocationStrategy.hpp"
 #include "umpire/strategy/DynamicPool.hpp"
-
 #include "umpire/util/io.hpp"
 
-#include "umpire/Replay.hpp"
-
 namespace umpire {
 
 static const char* env_name = "UMPIRE_REPLAY";
 
 Replay::Replay() : m_replayUid(getpid())
 {
   char* enval = getenv(env_name);
-  bool enable_replay = ( enval != NULL );
+  bool enable_replay = (enval != NULL);
 
-  replayEnabled =  enable_replay;
+  replayEnabled = enable_replay;
 }
 
-void Replay::logMessage( const std::string& message )
+void Replay::logMessage(const std::string& message)
 {
-  if ( !replayEnabled )
-    return;   /* short-circuit */
+  if (!replayEnabled) return; /* short-circuit */
 
   umpire::replay() << message;
 }
 
-bool Replay::replayLoggingEnabled()
-{
-  return replayEnabled;
-}
+bool Replay::replayLoggingEnabled() { return replayEnabled; }
```

Comment:
Same comment as above.  I would prefer:
```suggestion
bool Replay::replayLoggingEnabled()
{
  return replayEnabled;
}
```